homeassistant/components/statistics/sensor.py

```diff
@@ -376,6 +377,26 @@ def _derive_unit_of_measurement(self, new_state: State) -> str | None:
     @property
     def device_class(self) -> Literal[SensorDeviceClass.TIMESTAMP] | None:
```

Comment:
typing must now be adjusted... it can be any SensorDeviceClass

Reply:
Great catch! Fixed